versions-4.3.x.cfg

```diff
@@ -16,9 +16,9 @@ test-eggs =
 [versions]
 # XXX: https://github.com/collective/collective.cover/issues/492
```

Comment:
Maybe instead of doing what I said in the last command, put a `# Remember to edit README.rst if you change any dependency here`.

Reply:
I think that's not necessary at this time.